frontend/src/App.tsx

```diff
@@ -1,26 +1,23 @@
 import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import Home from "./components/Home/home";
+import Login from "./components/Login/login";
```

Comment:
I haven't said this before but component files should have a capital letter as well, so change the home and login files to Home and Login. Since we'll be using AWS Authentication and it should take care of the whole authentication process for us, I don't think we'll need the login code. You can remove that for now, we just need the page and some text